src/lib.rs

```diff
@@ -92,13 +93,24 @@ impl Default for Config {
 }
 
 impl Config {
+    /// Replace a path with a pattern if it appears in stderr.
```

Comment:
It's replaced with `replacement`, not with a pattern, right?

```suggestion
    /// Add filter to replace given path if it appears in stderr.
```